server/pom.xml

```diff
@@ -38,7 +38,28 @@
         <artifactId>gson</artifactId>
         <version>2.8.6</version>
     </dependency>
-
+  
+    <!-- Google API Client -->
+    <dependency>
+      <groupId>com.google.api-client</groupId>
+      <artifactId>google-api-client</artifactId>
+      <version>1.30.9</version>
+    </dependency>
+ 
+   <!-- Youtube API -->
+    <dependency>
+      <groupId>com.google.apis</groupId>
+      <artifactId>google-api-services-youtube</artifactId>
+      <version>v3-rev20200618-1.30.9</version>
+    </dependency>
+    
+    <!-- Google OAuth API -->
+    <dependency>
+      <groupId>com.google.apis</groupId>
+      <artifactId>google-api-services-oauth2</artifactId>
+      <version>v2-rev20200213-1.30.9</version>
+    </dependency>
+    
```

Comment:
Why did you include these dependencies? It doesn't seem like you are using them (instead you're just making all of the HTTP requests directly).

Reply:
oh right let me get rid of them XD I added them cause I was going to use the java Youtube API package, but I'm not doing that anymore I'm just doing http requests